seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumRun.scala

```diff
@@ -171,13 +178,14 @@ private[selenium] object SeleniumRun {
   private def maybeCleanupDriver(d: WebDriver, config: SeleniumJSEnv.Config) =
     if (!config.keepAlive) d.close()
 
-  private def htmlPage(scripts: Seq[java.net.URL]): String = {
-    val scriptTags =
-      scripts.map(path => s"<script src='${path.toString}'></script>")
+  private def htmlPage(scripts: Seq[java.net.URL], modules: Seq[java.net.URL]): String = {
+    val scriptTags = scripts.map(path => s"<script src='${path.toString}'></script>")
+    val moduleTags = modules.map(path => s"<script type='module' src='${path.toString}'></script>")
```

Comment:
This does not preserve the loading order of scripts v.s. modules. The order should be preserved.